libtrap/src/ifc_file.c

```diff
@@ -500,12 +542,8 @@ int create_file_recv_ifc(trap_ctx_priv_t *ctx, const char *params, trap_input_if
 
 void open_next_file_wrapper(void *priv)
```

Comment:
Do we still need open_next_file_wrapper()? It currently contains only a one-line if...

Reply:
Wrapper is only for the purpose when disconnect client happens in libtrap. The function (wrapper) must return void and take one argument void * (context). The wrapper is only called from trap.c.
line 880: ifc->disconn_clients = open_next_file_wrapper;